backend/docs/11_Exporting_result_data.md

```diff
@@ -377,3 +377,12 @@ You will be presented with a screen that lets you choose the fields that you wan
 ## The database structure
 [![muscle_db_visualized](https://github.com/user-attachments/assets/f0d97029-ec49-4bcc-8c84-8ed35b0937c0)](https://github.com/user-attachments/assets/f0d97029-ec49-4bcc-8c84-8ed35b0937c0)
 [*click to open the image to enable zoom functionality*](https://github.com/user-attachments/assets/f0d97029-ec49-4bcc-8c84-8ed35b0937c0)
+
+## Note on Timestamps
+
+All timestamps in JSON exports are in UTC time. This ensures consistency across different systems and time zones. Please also note that the currently configured timezone of the server is 'Europe/Amsterdam'. You can change these settings in the `base_settings.py` file in the `backend/aml` directory. Look for:
```

Comment:
Hm, it may not be so clear to outside collaborators what "the server" is. Perhaps something like this is clearer?
```suggestion
All timestamps in JSON exports are in UTC time. This ensures consistency across different systems and time zones. Please also note that the currently configured timezone for this application is 'Europe/Amsterdam'. If you would like to fork and spin your own instance with another timezone setting, you can change the `base_settings.py` file in the `backend/aml` directory. Look for:
```